src/pages/en/guides/migrate-to-astro/from-wordpress.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Migrating from WordPress
+description: Tips for migrating an existing WordPress project to Astro
+layout: ~/layouts/MigrationLayout.astro
+stub: true
+framework: WordPress
+---
+
+[WordPress](https://wordpress.org) is an open-source, personal publishing system built on PHP and MySQL.
+
+
+:::tip
+You can [use WordPress as a headless CMS for your Astro project](/en/guides/cms/wordpress/). Follow our guide to use your existing WordPress content in a new Astro project.
+:::
+
+## Key Similarities between WordPress and Astro
+
+WordPress and Astro share some similarities that will help you migrate your project:
+
+- Both WordPress and Astro are ideal for content-focused sites like blogs and support writing your content in Markdown. Although the process for adding new content is different, writing in Markdown files for your Astro blog should feel familiar if you have used Markdown syntax in your WordPress editor.
+
+- Both WordPress and Astro encourage you to think about the design of your site in "blocks" (components). In Astro you will probably write more of your own code to create these blocks rather than repy on pre-built plugins. But thinking about the individual pieces of your site and how they are presented on the page should feel familiar.
```

Comment:
```suggestion
- Both WordPress and Astro encourage you to think about the design of your site in "blocks" (components). In Astro you will probably write more of your own code to create these blocks rather than rely on pre-built plugins. But thinking about the individual pieces of your site and how they are presented on the page should feel familiar.
```